polaris.shopify.com/pages/examples/index-table-with-subheaders.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  IndexTable,
+  LegacyCard,
+  Text,
+  HorizontalStack,
+  Button,
+} from '@shopify/polaris';
+import {ChevronUpMinor} from '@shopify/polaris-icons';
+import React, {Fragment} from 'react';
+import {withPolarisExample} from '../../src/components/PolarisExampleWrapper';
+
+function WithSubHeadersExample() {
+  const customers = [
+    {
+      id: '3411',
+      url: '#',
+      name: 'Mae Jemison',
+      location: 'Decatur, USA',
+      orders: 20,
+      amountSpent: '$2,400',
+    },
+    {
+      id: '2561',
+      url: '#',
+      name: 'Ellen Ochoa',
+      location: 'Los Angeles, USA',
+      orders: 30,
+      amountSpent: '$140',
+    },
+    {
+      id: '2561',
+      url: '#',
+      name: 'Ellen Ochoa',
+      location: 'Los Angeles, USA',
+      orders: 30,
+      amountSpent: '$140',
+    },
+    {
+      id: '2561',
+      url: '#',
+      name: 'Ellen Ochoa',
+      location: 'Los Angeles, USA',
+      orders: 30,
+      amountSpent: '$140',
+    },
+  ];
+  const resourceName = {
+    singular: 'customer',
+    plural: 'customers',
+  };
+
+  const rowMarkup = customers.map(
+    ({id, name, location, orders, amountSpent}, index) => (
+      <Fragment key={id}>
+        <IndexTable.Row
+          header
+          id={id}
+          position={index}
+          selected={index === 3 ? 'indeterminate' : index === 0}
+          disabled={index === 1}
+        >
+          <IndexTable.Cell colSpan={3}>Header</IndexTable.Cell>
+          <IndexTable.Cell>
+            <HorizontalStack align="end">
+              <Button icon={ChevronUpMinor} plain />
```

Comment:
(I think the a11y test CI check is complaining about the icon only buttons needing aria labels, so just update this sudo suggestion with whatever makes sense for this example)

```suggestion
              <Button accessibilityLabel="description of what this means in the context it's used" icon={ChevronUpMinor} plain />
```